go/service/tracker_loader.go

```diff
@@ -12,11 +12,17 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+type trackerLoaderContext struct {
+	ctx    context.Context
+	cancel context.CancelFunc
```

Comment:
could just store the cancel func instead of this struct right?

Reply:
done